src/lib.rs

```diff
@@ -598,8 +598,8 @@ pub struct CertificateParams {
 impl Default for CertificateParams {
 	fn default() -> Self {
 		// not_before and not_after set to reasonably long dates
-		let not_before = date_time_ymd(1975, 01, 01);
-		let not_after = date_time_ymd(4096, 01, 01);
+		let not_before = date_time_ymd(1975, 1, 1);
```

Comment:
01 is in accordance with how dates are commonly written. I like it better that way.

Reply:
I'm confused because `01` is an octal number, even when `01 == 1`

Reply:
Good point, but Rust has no octal literals.

Reply:
Oh, you're right. In C `01` is actutal an octal literal. A clippy hint solves it, too.